biz.aQute.bndlib/src/aQute/bnd/build/Workspace.java

```diff
@@ -1801,4 +1817,12 @@ public OnWorkspace on(String ownerName) {
 		return notifier.on(ownerName);
 	}
 
+	public Collection<? extends Capability> getAvailableWorkspaceExtensions() {
```

Comment:
Do these really need to be public API? For the most part we should try hard to just use private or default accessibility for new methods. Making then public or protected means we are making API we need to support in the long term.

Reply:
you are right, this is unnecessary, as every run will go through the repositories anyway an.